workspace.json

```diff
@@ -1710,6 +1710,34 @@
         }
       }
     },
+    "application-templates-party-letter": {
+      "root": "libs/application/templates/party-letter",
+      "sourceRoot": "libs/application/templates/party-letter/src",
+      "projectType": "library",
+      "schematics": {},
+      "architect": {
+        "lint": {
+          "builder": "@nrwl/linter:lint",
+          "options": {
+            "linter": "eslint",
+            "tsConfig": [
+              "libs/application/templates/party-letter/tsconfig.lib.json",
+              "libs/application/templates/party-letter/tsconfig.spec.json"
+            ],
+            "exclude": [
+              "**/node_modules/**",
+              "!libs/application/templates/party-letter/**/*"
+            ]
+          }
+        },
+        "extract-strings": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "yarn ts-node -P libs/localization/tsconfig.lib.json libs/localization/scripts/extract 'libs/application/core/src/**/*.{js,ts,tsx}'"
```

Comment:
Wrong path, should point to your partyLetter AS directory or else to messages.ts if you will only use one file for messages

Reply:
Fixed